### PR TITLE
Staging/40

### LINKED ISSUE #40: Lodging API should support images.
Create image controller.
Connect to Azure Cloud Blob Storage.
Image controller needs to have integration with Azure Cloud Blob Storage.
Seed images into storage.

### PR DESCRIPTION
**Pull request by Jonny-Code**
resolves #40

### CODE REVIEW

File: aspnet/RVTR.Lodging.UnitTesting/Tests/ImageModelTest.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        {
+          imageId = 1,
+          lodgingId = 1,
+          imageUrl = "https://www.cbc.ca/parents/content/imgs/weird-kids-youtube.jpg",
+        }
+      }
+    };
+
+    /// <summary>
+    ///test data for validate of validation to image model table validationing
```
Comment (markmoorerev) [inline aspnet/RVTR.Lodging.UnitTesting/Tests/ImageModelTest.cs:26]: clarification needed

File: aspnet/RVTR.Lodging.UnitTesting/Tests/ImageModelTest.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    public void Test_Create_ImageModel(ImageModel image)
+    {
+      var validationContext = new ValidationContext(image);
+      var actual = Validator.TryValidateObject(image, validationContext, null, true);
+
+      Assert.True(actual);
+    }
+
+    /// <summary>
+    ///Doing validation to image model validating work of the validationing validater in the validator
```
Comment (markmoorerev) [inline aspnet/RVTR.Lodging.UnitTesting/Tests/ImageModelTest.cs:40]: clarification needed

File: aspnet/RVTR.Lodging.ObjectModel/Models/LodgingModel.cs [hunk 3]
```diff
@@ -1,5 +1,6 @@
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
+using System.Reflection;
```
Comment (escalonn) [inline aspnet/RVTR.Lodging.ObjectModel/Models/LodgingModel.cs:3]: doesn't belong

File: aspnet/RVTR.Lodging.ObjectModel/Models/ImageModel.cs [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+namespace RVTR.Lodging.ObjectModel.Models
+{
+    public class ImageModel : IValidatableObject
+    {
+        /// <summary>
+        /// The public key Id for each image url 
+        /// </summary>
+        [Key]
+        public int imageId { get; set; }
```
Comment (escalonn) [inline aspnet/RVTR.Lodging.ObjectModel/Models/ImageModel.cs:12]: need correct naming convention

**Review by escalonn — CHANGES_REQUESTED** [review]
add images to seed data
either split issue 40 or dont make this PR resolve it

**Pull request closed without merging**